docs/learn/features.md

```diff
@@ -0,0 +1,36 @@
+# Features
+Ceramic is a decentralized, open source platform for creating, hosting, and sharing streams of data. With Ceramic's permissionless data streaming network, you can store streams of information and ever-changing files directly on the decentralized web – and share updates with anyone in the world. This page describes the most important features of the protocol.
+
+### **Mutable streams**
+Store, edit, and update continuous [streams](./glossary.md#ipfs) of content or ever-changing files using stateful data structures on [IPFS](./glossary.md#ipfs) (specifically [IPLD](./glossary.md#ipld)). The data structure of your streams can be fully customized to your specific use cases and needs.
+
+### **Immutable naming**
+Reference streams by a persistent identifier, called a [StreamID](./glossary.md#streamid), instead of needing to keep track of IPFS [CIDs](./glossary.md#cid) that change every time your content does.
+
+### **Custom functions**
+Write custom functions for processing updates to your stream's [state](./glossary.md#state). These functions are called [StreamTypes](./glossary.md#streamtypes), and they're deployed to a Ceramic [node](./glossary.md#streamid). StreamTypes ingest new updates and autonomously apply transformations to your stream – guaranteeing data consistency and integrity without needing to rely on an external source of compute logic or state management. Ceramic nodes come [prepackaged with common StreamTypes](../streamtypes/overview.md) making it easy to get started creating applications without needing to code your own.
+
+### **DID authentication**
+[Authenticate](./glossary.md#authentication) to and transact with streams using W3C-standard decentralized identities ([DIDs](./glossary.md#dids)). While every StreamType is able to define its own authentication requirements and mechanisms, DIDs are the most common. Notably DIDs can be controlled with one or more Web3/blockchain wallets, so your users can build up a unified cross-chain Web3 identity using the wallets they already have.
+
+### **Transparent history**
+Inspect, audit, or time travel through the complete history of any stream to see how its content has changed over time.
+
+### **Scalable consensus**
+Unlike blockchains or other DLT systems which have scalability limitations due to reliances on a single execution environment and global state, Ceramic takea a different approach to network scalability. On Ceramic, every stream maintains its own state and nodes independently process stream transactions, allowing for unbounded parallelization. This enables Ceramic to operate at worldwide data scale, which is orders of magnitude greater than the scale needed for decentralized finance.
+
+### **Host & deploy**
+Host and make your streams available over a decentralized, global, peer-to-peer network that is censorship-resistant and completely free of middlemen.
```

Comment:
The content in this section is good and feels like it could even come earlier on the page, but I'm not sure "Host & deploy" is the right heading for it?

Reply:
Changed to "Global Availability"